src/test/java/gov/healthit/chpl/aqa/pageObjects/SurveillanceReportPage.java

```diff
@@ -10,64 +10,128 @@
+    public static WebElement surveillanceCompletedCapVerification(final WebDriver driver) {
+        return driver.findElement(By.id("completed-cap-verification"));
     }
 
-    public static WebElement quarterlyReportFieldInput(final WebDriver driver, final String fieldInputId) {
-        return driver.findElement(By.xpath("//textarea[@id='" + fieldInputId + "']"));
+    public static WebElement surveillanceDirectionDeveloperResolution(final WebDriver driver) {
+        return driver.findElement(By.id("direction-developer-resolution"));
     }
 
-    public static WebElement saveQuarterlyReport(final WebDriver driver) {
-        return driver.findElement(By.xpath("//button[@id='quarterly-surveillance-save']"));
+    public static WebElement surveillanceLimitationsEvaluation(final WebDriver driver) {
+        return driver.findElement(By.id("limitations-evaluation"));
     }
 
-    public static WebElement editSurveillanceReport(final WebDriver driver, final String acbNameYearQuarter) {
-        return driver.findElement(By.xpath("//button[@id='act-" + acbNameYearQuarter + "']"));
+    public static WebElement surveillanceNonDisclosureEvaluation(final WebDriver driver) {
+        return driver.findElement(By.id("nondisclosure-evaluation"));
+    }
+
+    public static WebElement surveillanceOutcomeDropdown(final WebDriver driver) {
+        return driver.findElement(By.id("surveillance-outcome"));
+    }
+
+    public static WebElement surveillanceProcessTypeDropdown(final WebDriver driver) {
+        return driver.findElement(By.id("surveillance-process-type"));
+    }
+
+    public static WebElement surveillancePotentialCauses(final WebDriver driver) {
+        return driver.findElement(By.id("nonconformity-causes"));
+    }
+
+    public static WebElement surveillanceNonConformityNature(final WebDriver driver) {
+        return driver.findElement(By.id("nonconformity-nature"));
+    }
+
+    public static WebElement surveillanceStepsToEngage(final WebDriver driver) {
+        return driver.findElement(By.id("steps-to-engage"));
+    }
+
+    public static WebElement surveillanceStepsToSurveil(final WebDriver driver) {
+        return driver.findElement(By.id("steps-to-surveil"));
     }
 
     public static WebElement surveillancePageSubtitle(final WebDriver driver) {
         return driver.findElement(By.xpath("//h2[contains(text(),'Available reports')]"));
     }
 
-    public static WebElement initiateButton(final WebDriver driver, final String acbNameYearQuarter) {
-        return driver.findElement(By.xpath("//button[@id='initiate-" + acbNameYearQuarter + "']"));
+    public static WebElement surveillancePageTitle(final WebDriver driver) {
+        return driver.findElement(By.xpath("//h1[contains(text(),'CHPL Surveillance')]"));
+    }
+
+    public static WebElement transparencyDisclosureField(final WebDriver driver) {
+        return driver.findElement(By.xpath("//textarea[@id='transparency-disclosure']"));
```

Comment:
Was there a reason not to use "By.id"?

Reply:
I didn't add that element definition, just rearranged what was already there, but agree that it should be ID. I'll update that